docs/zh-cn/newbs_testing_debugging.md

```diff
@@ -1,5 +1,10 @@
 # 测试和调试
```

Comment:
This was restructured into 2 new places see https://github.com/qmk/qmk_firmware/blob/master/docs/newbs_testing_debugging.md